src/init/features/ailogic/index.ts

```diff
@@ -0,0 +1,149 @@
+import * as fs from "fs-extra";
+import { Config } from "../../../config";
+import { select, input, confirm } from "../../../prompt";
+import { Setup } from "../..";
+import { FirebaseError } from "../../../error";
+import {
+  SupportedPlatform,
+  detectAppPlatform,
+  buildProvisionOptions,
+  provisionAiLogicApp,
+  getConfigFilePath,
+  writeAppConfigFile,
+  extractProjectIdFromAppResource,
+} from "./utils";
+
+export interface AiLogicInfo {
+  appPlatform?: "android" | "ios" | "web";
+  appNamespace: string;
+  overwriteConfig?: boolean;
+}
+
+/**
+ * Ask questions for AI Logic setup via CLI
+ */
+export async function askQuestions(setup: Setup): Promise<void> {
+  // Ask for app platform
+  const platform = await select<"android" | "ios" | "web">({
+    message: "Which platform would you like to set up?",
+    choices: [
+      { name: "Android", value: "android" },
+      { name: "iOS", value: "ios" },
+      { name: "Web", value: "web" },
+    ],
+  });
+
+  // Ask for app namespace
+  let appNamespace: string;
+  if (platform === "android") {
+    appNamespace = await input({
+      message: "Enter your Android package name (e.g., com.example.myapp):",
+      validate: (input: string) => {
+        if (!input || !/^[a-zA-Z][a-zA-Z0-9_]*(\.[a-zA-Z][a-zA-Z0-9_]*)*$/.test(input)) {
+          return "Please enter a valid Android package name (e.g., com.example.myapp)";
+        }
+        return true;
+      },
+    });
+  } else if (platform === "ios") {
+    appNamespace = await input({
+      message: "Enter your iOS bundle ID (e.g., com.example.MyApp):",
+      validate: (input: string) => {
+        if (!input || !/^[a-zA-Z][a-zA-Z0-9_]*(\.[a-zA-Z][a-zA-Z0-9_]*)*$/.test(input)) {
+          return "Please enter a valid iOS bundle ID (e.g., com.example.MyApp)";
+        }
+        return true;
+      },
+    });
+  } else {
+    appNamespace = await input({
+      message: "Enter your web app name:",
+      validate: (input: string) => {
+        if (!input) {
+          return "Please enter a web app name";
+        }
+        return true;
+      },
+    });
+  }
+
+  // Ask about overwriting existing config files
+  const overwriteConfig = await confirm({
+    message: "Would you like to overwrite existing config files if they exist?",
+    default: false,
+  });
+
+  // Set up the feature info
+  if (!setup.featureInfo) {
+    setup.featureInfo = {};
+  }
+
+  setup.featureInfo.ailogic = {
+    appPlatform: platform,
+    appNamespace: appNamespace,
+    overwriteConfig: overwriteConfig,
+  };
+}
+
+/**
+ * AI Logic provisioning: auto-detects/creates project and app, provisions via API
```

Comment:
As I read through this more, I am worried that this is pushing beyond the scope of what a feature init should be doing. During init, we first ask you what project you want to use, and then ask what features. With this change, I could get the following confusing behavior:

- I run firebase init, and create projectA
- I select FAIL from the list of products, plus a few others
- The other products get init'd using projectA
- FAIL picks up projectB from the resource, and then tries to create projectB and enable FAIl there.

We might need to rethink init more fundamentally in order to correctly use this new API

Reply:
Completely removed the ability create new project/app in the implementation. The code now check for existing project/app and only set up and enable Firebase AI Logic for the app